explorer/build.gradle

```diff
@@ -13,7 +13,7 @@ repositories {
 }
 
 dependencies {
-    compile group: 'com.scalar-labs', name: 'scalardl-client-sdk', version: '1.1.0'
+    compile group: 'com.scalar-labs', name: 'scalardl-java-client-sdk', version: '2.0.4'
```

Comment:
```suggestion
    compile group: 'com.scalar-labs', name: 'scalardl-java-client-sdk', version: '2.0.5'
```
Just upgraded :)

Reply:
Thank you for review change.
SDK version updated in 135ba8e5358cdc9bfd02084406fd65ba68b3ddd4